src/web/layouts/ShowcaseLayout.tsx

```diff
@@ -411,6 +411,13 @@ export const ShowcaseLayout = ({ CAPI, NAV }: Props) => {
                 </ShowcaseGrid>
             </Section>
 
+            <Section padded={false} showTopBorder={false}>
+                <AdSlot
+                    asps={namedAdSlotParameters('merchandising-high')}
+                    className=""
```

Comment:
className not optional?

Reply:
Nope. There's a few possible refactors I can see in `AdSlot` but I tread with care in that area and if the win isn't clear then the risk seems, well, risky.

Reply:
Hmm, rereading that back makes me realise that we really do need to tackle this because if we don't it might get worse and get even harder and riskier to change. Shall add ticket.

Reply:
Here you go: https://trello.com/c/1wsSZq9W/1140-refactor-adslot

Reply:
nice 👍